filc/cmds.go

```diff
@@ -303,11 +307,16 @@ var retrieveFileCmd = &cli.Command{
 			output = cidStr
 		}
 
-		root, err := cid.Decode(cidStr)
+		noIPFS := cctx.Bool(flagNoIPFS.Name)
+		println("NO IPFS:", noIPFS)
```

Comment:
avoid using the builtin `println`, its a bit of a random hack in the language that shouldnt be there anymore